### PR TITLE
Updated create-a-custom-dictionary.md 

### PR DESCRIPTION
**Pull request by modery**
Updated /create-a-custom-dictionary.md to use a markdown table instead of an HTML table

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointServer/search/create-a-custom-dictionary.md [hunk 1]
```diff
@@ -77,52 +77,16 @@ The following table shows examples of supported and unsupported entries.
...
-<td><p>C7H1603</p></td>
-<td><p></p></td>
-</tr>
-</tbody>
-</table>
+| Supported | Not supported |
+| --- | --- |
+| dogfood | dog food |
+| 3# | #3 |
+| For#sale | For|sale |
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointServer/search/create-a-custom-dictionary.md:84]: @modery - please check the rich diff view, this line does not display correctly (escape character).

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: SharePoint/SharePointServer/search/create-a-custom-dictionary.md (hunk 1 above)
Comment (modery) [reply SharePoint/SharePointServer/search/create-a-custom-dictionary.md]: Added escape character to correct the problem

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**